packages/linter/src/rules/ModuleRuntimeUsed.ts

```diff
@@ -0,0 +1,27 @@
+import { Context } from "../index";
+import { Rule } from "../rule";
+
+export class ModuleRuntimeUsed extends Rule {
+  run({ $ }: Context) {
+    const isTransformed = $("html[transformed='self;v=1']").length > 0;
```

Comment:
Might make sense to extract this into a helper method. We'll need it more often.

Reply:
done